linera-service/src/client.rs

```diff
@@ -83,11 +90,17 @@ pub struct ClientWrapper {
 }
 
 impl ClientWrapper {
-    fn new(tmp_dir: Rc<TempDir>, network: Network, id: usize) -> Self {
+    fn new(tmp_dir: Rc<TempDir>, database: Database, network: Network, id: usize) -> Self {
+        let storage = match database {
+            Database::RocksDb => format!("rocksdb:client_{}.db", id),
```

Comment:
Can you turn this into a method in `impl Database { .. }` ?
I look forward to the replacement by proper configuration files (#993).